app/extensions/edm/__init__.py

```diff
@@ -88,6 +91,21 @@ class EDMManager(RestManager):
     def __init__(self, pre_initialize=False, *args, **kwargs):
         super(EDMManager, self).__init__(pre_initialize, *args, **kwargs)
 
+    def version_check(self):
+        edm_version = current_app.edm.get_dict('version.dict', None)
```

Comment:
surely at this point current_app.edm is the same as self?

Reply:
by jove, yes.   just a little too quick with cut-n-paste.  updated.